currency/forexprovider/fixer.io/fixer_test.go

```diff
@@ -13,7 +13,7 @@ var f Fixer
 
 var isSetup bool
 
-func setup(t *testing.T) {
+func setup(t *testing.T) { //nolint
```

Comment:
Please be more specific with your nolint usage. Here is an example from our codebase:
`//nolint:forcetypeassert // Not necessary from a pool`
we specify what we are nolinting and we provide a comment on why we are nolinting